### PR TITLE
Adding how to set an error with opentracing and java

### PR DESCRIPTION
**Pull request by l0k0ms**
### What does this PR do?

Adds info on how to set a span as error with opentracing in java

### Motivation

Support feedback

### Preview link

* https://docs-staging.datadoghq.com/gus/errors-opentracing-faq/tracing/advanced/opentracing/?tab=java#set-errors

### CODE REVIEW

**Comment by tylerbenson** [comment]
This is the specific way our backend supports, but OpenTracing has a different way of handling it (using span.log).  Which way makes more sense?

File: content/en/tracing/advanced/opentracing.md [hunk 1]
```diff
@@ -239,11 +238,39 @@ public class MyHttpRequestExtractAdapter implements TextMap {
...
+
+Then you need to set the error to `true` and add *at least* the `error.msg`, `error.type`, and `error.stack` tag to your span.
+
+```java
+    final Span span = GlobalTracer.get().activeSpan();
+    if (span != null) {
+      Tags.ERROR.set(span, true);
+      span.setTag("error.msg", "<ERROR_MESSAGE>");
+      span.setTag("error.type", "<EXCEPTION_NAME>");
+      span.setTag("error.stack", "<ERROR_STACK>");
```
Comment (tylerbenson) [inline content/en/tracing/advanced/opentracing.md]: Replace these with
```java
span.log(Collections.singletonMap(Fields.ERROR_OBJECT, ex));
```

to better align with OpenTracing: https://github.com/opentracing/opentracing-java

File: content/en/tracing/advanced/opentracing.md [hunk 2]
```diff
@@ -239,11 +238,37 @@ public class MyHttpRequestExtractAdapter implements TextMap {
   }
 }
 ```
+
 Notice the above examples only use the OpenTracing classes. Check the [OpenTracing API][1] for more details and information.
 
+#### Set errors
+
+To customize an error associated to one of your span, you must import the `io.opentracing.Span`, `io.opentracing.tag.Tags`, and `io.opentracing.util.GlobalTracer` library in the method where the error occurs:
```
Comment (ruthnaebeck) [inline content/en/tracing/advanced/opentracing.md]:
```suggestion
To customize an error associated to one of your spans, import the `io.opentracing.Span`, `io.opentracing.tag.Tags`, and `io.opentracing.util.GlobalTracer` libraries into the method where the error occurs:
```

File: content/en/tracing/advanced/opentracing.md [hunk 3]
```diff
@@ -239,11 +238,37 @@ public class MyHttpRequestExtractAdapter implements TextMap {
...
+
+```java
+import io.opentracing.Span;
+import io.opentracing.tag.Tags;
+import io.opentracing.util.GlobalTracer;
+```
+
+Then you need to set the error to `true` and add *at least* the `error.msg`, `error.type`, and `error.stack` tag to your span.
```
Comment (ruthnaebeck) [inline content/en/tracing/advanced/opentracing.md]:
```suggestion
Then, set the error to `true` and add *at least* the `error.msg`, `error.type`, and `error.stack` tags to your span.
```

File: content/en/tracing/advanced/opentracing.md [hunk 4]
```diff
@@ -239,11 +238,37 @@ public class MyHttpRequestExtractAdapter implements TextMap {
...
+
+```java
+    final Span span = GlobalTracer.get().activeSpan();
+    if (span != null) {
+      Tags.ERROR.set(span, true);
+      span.log(Collections.singletonMap(Fields.ERROR_OBJECT, ex));
+    }
+```
+
+**Note**: Any relevant error metadata explained in [the Trace View docs][4] can also be added.
```
Comment (ruthnaebeck) [inline content/en/tracing/advanced/opentracing.md]:
```suggestion
**Note**: Any relevant error metadata explained in the [Trace View docs][4] can also be added.
```

File: content/en/tracing/advanced/opentracing.md [hunk 5]
```diff
@@ -239,11 +238,37 @@ public class MyHttpRequestExtractAdapter implements TextMap {
...
+If the current span isn't the root span, mark it as an error by using the `dd-trace-api` library to grab the root span with `MutableSpan` and use `setError(true)` (See [source code][5]).
```
Comment (ruthnaebeck) [inline content/en/tracing/advanced/opentracing.md]:
```suggestion
If the current span isn't the root span, mark it as an error by using the `dd-trace-api` library to grab the root span with `MutableSpan`, then use `setError(true)`. See the [source code][5] for more details.
```

**Pull request merged**